tensorboard/plugins/core/core_plugin.py

```diff
@@ -568,8 +568,10 @@ def fix_flags(self, flags):
     elif flags.host is not None and flags.bind_all:
       raise FlagsError('Must not specify both --host and --bind_all.')
 
-    if flags.path_prefix.endswith('/'):
-      flags.path_prefix = flags.path_prefix[:-1]
+    flags.path_prefix = flags.path_prefix.rstrip('/')
+    if flags.path_prefix and not flags.path_prefix.startswith('/'):
+      raise FlagsError(
+          'Path prefix must start with slash, but got: %r.' % flags.path_prefix)
```

Comment:
unless the test is already present, consider adding a test on this.

Reply:
Thanks for the suggestion. Done.